docker-compose.yml

```diff
@@ -432,3 +432,61 @@ services:
     image: prom/pushgateway
     ports:
       - 9091:9091
+
+  priority-worker:
```

Comment:
I think for the main repo, we should not create a priority-worker. Might be good to put this in the `docker-compose-multiworker.yaml` instead.